blazor/datagrid/how-to/using-different-edit-mode-for-add-edit-action.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Using different edit mode for add and edit action in Blazor DataGrid | Syncfusion
+description: Learn here all about using different edit mode for add and edit action in Syncfusion Blazor DataGrid component and more.
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# Use different EditMode for Add/Edit action
+
+You can customize the EditMode for add and edit actions as `Dialog` or `Normal` based on the RequestType value in [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnActionBegin) event.
```

Comment:
Customize the EditMode for add and edit actions as `Dialog` or `Normal` based on the RequestType value in the [OnActionBegin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnActionBegin) event.